### ISSUE TITLE
Support non-legacy Raspberry Pi OS

### ISSUE THREAD

**Issue by jneuhauser** [pibooth/pibooth#540]
**Is your feature request related to a problem? Please describe.**
Buster is end of life on June 30th, 2024.

**Describe the solution you'd like**
Support Bookworm and Bullseye

**Describe alternatives you've considered**
Running Buster until June 30th, 2024.

How about to track the required steps here in this ticket?

**Comment by nsty** [comment]
Did you try it? I'm pretty sure I had it running on Bookworm without modification. I think the docs are just outdated.

**Comment by omgsideburns** [comment]
I don't believe the current build release (2.0.8?) is working on bookworm as it's still utilizing the old camera libraries. 

I haven't dug through the 3.0 code to see if they've been implementing the new libraries yet either.

**Comment by nsty** [comment]
You're right, it wasn't Bookworm. I just took a look. My photo booth was on Bullseye with Pibooth release 2.0.8. So this was definitively running without a hassle. Still, the issue is valid as Bookworm is the current release.

**Comment by werdeil** [comment]
As written in https://github.com/pibooth/pibooth/issues/586#issuecomment-2169176932

> Debian 12 and all Python 3.11+ OS are indeed introducing a change on the way Python projects are installed by encouraging the use of virtualenv. We will need to update the install instruction to add something about it (either use virtualenv or the --break-system-packages option)

**Comment by rogusdev** [comment]
Is there any summary of the known blockers? In case someone was to attempt to resolve this on their own?
